src/cmds/init.py

```diff
@@ -366,6 +366,12 @@ def chia_init(root_path: Path):
         "pool.xch_target_address",
         "ALERTS_URL",
         "CHIA_ALERTS_PUBKEY",
+        "pool.xch_target_address",
+        "full_node.xch_target_address",
+        "farmer.farmer_target",  # These were buggy values in rc7
+        "farmer.pool_target",  # These were buggy values in rc7
+        "pool.pool_target",  # These were buggy values in rc7
+        "farmer.farmer_target",  # These were buggy values in rc7
```

Comment:
is `"farmer.farmer_target"` duplicated intentionally?

Reply:
no, thanks!

Reply:
👍